src/commcare_cloud/ansible/restart_webworker_services.yml

```diff
@@ -1,6 +1,6 @@
 - name: Rolling restart webworker services - ({{ webworker_hosts }})
   hosts: "{{ webworker_hosts }}"
-  serial: ["10%", "30%", "50%", "100%"]
+  serial: ["10%", "40%", "75%", "100%"]
```

Comment:
I believe it was previously
```
["10%", "40%", "100%"]
```
just looking at https://github.com/dimagi/commcare-cloud/pull/6094/files#diff-3d7593c6d6f4d6c05a0a5c9a194301e3eaec3cc436f730dfa64b233fa5102054L3.

Also based on the logic of what ansible does with these batch sizes, I think actually both what you have here and what we had before will result in exactly the same behavior: three batches of 10%, then 40%, then 50%.

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_strategies.html#rolling-update-batch-size describes this. Since after doing 10% and then 40%, there is 50% remaining, whether the next batch size is 75% or 100% (both larger than 50%), it will just do all of the remaining 50%. Then it's done so if there's another batch size in the list (100%) then it won't get used because it's already done by that point.

Reply:
I guess we could revert to the original numbers now since we have fixed the underlying problem.
10-40-100 or 10-40-50 are both okay and we don't need to do this in more than 3 batches

Reply:
Thanks @dannyroberts.
I think I compared it to previous commit's changes and got the values from there.  I forgot that the values were changed twice.